NetCasbin/InternalEnforcer.cs

```diff
@@ -461,6 +471,14 @@ private void NotifyPolicyChanged()
 
         private async Task NotifyPolicyChangedAsync()
         {
+            if (autoCleanEnforceCache && EnforceCache is not null)
```

Comment:
when this method is invoked? if one policy is changed why we need to clear all the cache.

Reply:
The cache may return a wrong result If a policy is changed.

Reply:
If so I think the cache hit rate would be low.